src/renderer/mod.rs

```diff
@@ -718,12 +718,9 @@ impl QuadRenderer {
     }
 
     pub fn reload_shaders(&mut self, size: PhysicalSize) {
-        warn!("Reloading shaders ...");
+        warn!("Reloading shaders");
         let program = match ShaderProgram::new(size) {
-            Ok(program) => {
-                warn!(" ... OK");
-                program
-            }
```

Comment:
This should not be removed. But since Rust doesn't just randomly fail, I think a better approach would be to put two messages. One in the `Ok` and one in the `Err` case.

So like this:
```
Ok(_) => {
    info!("Successfully reloaded shaders");
},
Err(_) => {
    warn!("Unable to reload shaders: {}", err);
}